api/apps/api/src/modules/blm/values/blm-values.module.ts

```diff
@@ -1,20 +1,15 @@
 import { Module } from '@nestjs/common';
 
-import { ProjectBlmRepository } from './project-blm-repository';
-import { TypeormProjectBlmRepository } from './typeorm-project-blm-repository';
+import { ProjectBlmRepository } from '@marxan-api/modules/blm';
+import { TypeormProjectBlmRepository } from './repositories/typeorm-project-blm-repository';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { ProjectBlm } from '@marxan-api/modules/blm/values/repositories/project-blm.api.entity';
+import { MemoryProjectBlmRepository } from '@marxan-api/modules/blm/values/repositories/memory-project-blm-repository';
+import { ProjectBlmRepositoryToken } from '@marxan-api/modules/blm/values/repositories/project-blm-repository';
+import { ProjectBlmRepositoryModule } from '@marxan-api/modules/blm/values/repositories/project-blm-repository.module';
```

Comment:
minor: would be good to keep things that are related to this very module as relative imports - just for a sake of reading brevity